rpachallenge/readme.md

```diff
@@ -0,0 +1,89 @@
+# Clicknium Automation Sample Solution - RPA challenge: input forms
+
+This is a sample of RPA challenge with [clicknium](https://www.clicknium.com/) web automation.
+The goal of this challenge is to create a workflow that will input data from a spreadsheet into the form fields on the screen.
+The fields will change position on the screen after every submission throughout 10 rounds thus the workflow must correctly identify where each spreadsheet record must be typed every time.
```

Comment:
The fields will change their positions on the screen after each submission throughout 10 rounds, therefore the workflow must correctly identify the position where each spreadsheet will be recorded and typed every time.